### DISCUSSION TITLE
Look into imdone

### DISCUSSION THREAD

**Discussion by liamcain** [tgrosinger/slated-obsidian#8] [Ideas]
Someone on Discord mentioned https://imdone.io/ and useful to draw inspiration from! Task management powered by markdown.

**Comment by tgrosinger** [comment]
Thanks for the link! It has some interesting ideas, but it seems to be very focused around a separate file for each card in the kanban board. I'm not sure that will work well withe the daily note system that I am trying to integrate with.

I do see a future where some kanban features are added to this plugin though, it will just need to be adapted from this to operate on the per-task basis.

**Comment by argenos** [comment]
Not sure if this falls in the scope of what you're trying to do, but I'd love having a "horizon view" similar to this one: https://timestripe.com/

**Comment by tgrosinger** [comment]
@argenos that tool has some really neat features. I am just getting started on the Slated Task View, and I think there could be room for some of those features. We already have weekly notes.  @liamcain what do you think about adding monthly and yearly notes to the Calendar and obsidian-daily-notes-interface?

**Reply by argenos** [reply]
Indeed! I've been moving my weekly note to the sidebar, which already feels like moving in that direction! It's still very manual though, so some UI improvements to change time granularity on the sidebar/its own view would be really neat.

**Reply by liamcain** [reply]
> @liamcain what do you think about adding monthly and yearly notes to the Calendar and obsidian-daily-notes-interface?

That's the plan!
